DTIPrep/DTIPrepRegister.pl

```diff
@@ -1127,12 +1118,12 @@ sub getToolName {
 =head3 getPipelineDate($file, $data_dir, $QCReport)
 
 Fetches the date at which the DTIPrep pipeline was run either in the processed
-MINC file's header or in the QCReport.
+MINC file's header or in the QC text report.
 
 INPUTS:
-  - MINC or QC report to look for tool information
-  - data_dir from the C<Config> table
-  - QC report created when C<$file> was created
+  - $file    : MINC or QC text report to look for tool information
+  - $data_dir: data directory (e.g. C</data/$PROJECT/data>)
```

Comment:
just curious, when do we have to escape `$Project` with a backslash `\` and when do we not have to?

Reply:
the escape is need when it is within a string variable within the code. For the documentation between `=pod` and `=cut` or comments, escaping is not needed, since those lines are not executed by the script. Hope this helps clarify. If not, maybe we could look at it together and I'll try to explain it better :).